src/components/presentational/Chat/Chat.css

```diff
@@ -0,0 +1,75 @@
+.mdhui-chat {
+    background: var(--mdhui-background-color-0);
+    font-size: 14px;
+}
+
+.mdhui-chat .mdhui-chat-log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    min-height: 20vh;
+}
+
+.mdhui-chat p,
+.mdhui-chat ul {
+    margin: .25rem 0;
+    word-break: break-word;
+    color: var(--mdhui-text-color-1);
+}
+
+.mdhui-chat input,
+.mdhui-chat button {
+    min-height: var(--mdhui-touch);
+    min-width: var(--mdhui-touch);
+    border: 0;
+    border-radius: var(--mdhui-card-border-radius);
```

Comment:
No need for border radius here it seems?

Reply:
Looks fine both with it and without? Are our outlines usually without rounded corners?

Reply:
The outline should match the container which is a rectangle, the rounded outline only appears when you focus which feels awkward